### ISSUE TITLE
Can't access wsl in File Explorer + Possible Fix

### ISSUE THREAD

**Issue by Kipura** [microsoft/WSL#9010]
### Version

Windows 11 22H2

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.68.1

### Distro Version

Ubuntu 20.04

### Other Software

docker desktop

### Repro Steps

Fresh Windows Install
Install wsl
install docker desktop
restart pc

### Expected Behavior

Accessing Linux in File Explorer works fine
![image](https://user-images.githubusercontent.com/76932863/195497418-cae0f732-b0ea-4af2-9d35-aabb04f9646c.png)

### Actual Behavior

Can't access the linux tab in file explorer
![image](https://user-images.githubusercontent.com/76932863/195497374-55bc68ee-771e-4650-b61f-5df4bd06a2db.png)

**Comment by Kipura** [comment]
I found a workaround. The address now is \\wsl$ probably because of an update. I had to go into regedit: Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Classes\CLSID{B2B4A4D1-2754-4140-A2EB-9A76D9D7CDC6}\Instance\InitPropertyBag

And change the ResName from \\wsl.localhost to \\wsl$ For that i had to take full ownership of the {B2B4A4D1-2754-4140-A2EB-9A76D9D7CDC6} folder and give me full access rights so i could change ResName. Now it's working fine.

I posted this as an issue because this probably should get fixed nonetheless.
